site-scrapers/CVSPharmacy/index.js

```diff
@@ -54,7 +54,8 @@ module.exports = async function GetAvailableAppointments(browser) {
             name: `${site.name} (${city})`,
             hasAvailability: responseLocation.status !== "Fully Booked",
             availability: {},
-            timestamp: timestamp,
+            timestamp: new Date(),
+            siteTimestamp: timestamp,
```

Comment:
This is fine.  I did notice that the "Hannaford" code is 
`timestamp: moment().format(),`

It was just the first one I happened to check for reference.  Everything else looks like `new Date()`

Reply:
Watch me throw another dart against merge commits. Hannaford's choice came in on a109b204566df6908d2d99b01f12952b7b33d0bf which was via #99 which was actually from 5ffce27f2590562b693c25a15eebbc086ed507cb which changed some other date math in `lib/RxTouch.js`.

@tkjones117 this was your change, was there a good reason to make Hannaford return the timestamp in a different way than all other scrapers.

(Of course, I still think that scrapers should not be allowed to return timestamps if they don't have special knowledge [umm, like CVS before this PR] and that `main.js` should be adding it in to the returned data in all cases, but at this point I'm just whining.)

Reply:
I used `moment().format()` in Hannaford and StopAndShop because of a request from Liv that we convert to using moment(). 
dug up the convo here: https://github.com/livgust/covid-vaccine-scrapers/pull/99#discussion_r584373250

Does the difference between Date() and moment() cause any difference in experience on the frontend?

Reply:
It doesn't make a difference. I like moment WAY better but it is in maintenance mode :(